lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -36,6 +36,8 @@ class DetailsRenderer {
         return this._renderText(details);
       case 'cards':
         return this._renderCards(/** @type {!DetailsRenderer.CardsDetailsJSON} */ (details));
+      case 'code':
```

Comment:
We should land the table renderer so I can use it in this PR :) Then we can make a generic `code` renderer.